src/types/payload.ts

```diff
@@ -95,8 +97,18 @@ export interface PlayedTileInteractionPayload {
 }
 
 export interface InteractionSuccessPayload {
+  connectionId?: string;
   playedTiles: string[];
   meldType: string;
   skipInteraction: boolean;
-  connectionId?: string;
+}
+
+export interface WinningTilesPayload {
+  connectionId: string;
+  tiles: string[];
+}
+
+export interface UpdateGameStatePayload {
+  dealer: number;
+  wind: number;
```

Comment:
maybe consider adding points to game state and payload?

Reply:
I'll probably add points to gamestate in another PR.

Reply:
Leave out points for now. We will discuss how to deal with it on Thursday

Reply:
We will still be passing in points. Interface of it TBD

Reply:
ok